pkg/provider/cloud/kubevirt/provider.go

```diff
@@ -72,8 +74,26 @@ func (k *kubevirt) ValidateCloudSpec(spec kubermaticv1.CloudSpec) error {
 	return nil
 }
 
-func (k *kubevirt) InitializeCloudProvider(c *kubermaticv1.Cluster, p provider.ClusterUpdater) (*kubermaticv1.Cluster, error) {
-	return c, nil
+func (k *kubevirt) InitializeCloudProvider(cluster *kubermaticv1.Cluster, update provider.ClusterUpdater) (*kubermaticv1.Cluster, error) {
+	return k.reconcileCluster(cluster, update)
```

Comment:
Normally, the `InitializeCloudProvider` should be a no-op for cluster that have already been initialized. I can see that this is not talking to any cloud API, but I am still a bit worried that it might be too slow to reconcile KubeVirt clusters basically on every single cluster reconciliation.

Reply:
Nevermind, the InitializeCloudProvider isn't even used for providers that have a Reconcile function, so it doesn't matter.